app/Http/Services/ApiServices/HrmsSecretKeyService.php

```
@@ -0,0 +1,238 @@
+                [
+                    'first_name',
+                    'middle_name',
+                    'family_name',
+                    'nick_name',
+                    'current_position',
+                    'digital_signature',
+                    'created_at',
+                    'updated_at',
+                    'deleted_at',
+                ]
+            );
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\SetupEmployees' in method 'syncEmployees'. (undefined)

(StaticAccess)